### ISSUE TITLE
'Run last run' or 'Restart' feature for Sublime Text Editor

### ISSUE THREAD

**Issue by redstoneleo** [DamnWidget/anaconda#390]
I am using Sublime Text Editor for Python development, one inconvenience I've met with like this: For example, I have two Python script, one is the launch entry named A.py and another is B.py, I write most of the code in B.py, I import B.py as a module in A.py, each time when I need to test whether my program work as expected , then I have to switch to A.py, then run it. After repeat this operation once and once again, I get tired enough.

I've ever used some IDEs such as PyCharm, Eclipse and Eric6, if you run a script last time and this time you want run it again , they all have a feature named 'Run last run' or 'Restart', this is a helpful shortcut, can I run A.py without switching to the tab it within when using Sublime Text Editor? any solutions ?

N.B. I don't want solve this inconvenience by changing my code.

**Comment by dzosz** [comment]
Good idea. Until it's implemented you could replace $file argument with your main executable filename.
Either create your own sublime build system:
> "cmd": ["python3.4", "-u", "main.py"]

Or save project file with Anaconda Python Builder. In the same place you specify virtual environment, add your filename.
Before:
> "shell_cmd": "python -u \"$file\""

After:
> "shell_cmd": "/home/yourname/.virtualenvs/project/bin/python -u main.py"

**Comment by redstoneleo** [comment]
@dzosz Thanks for your reply, are you going to implement the feature ?
